app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connection = sqlite3.connect("cinema_sqlite3_db")

```

Comment:
sqlite is file-based datbase, so you should have some file like cinema.sqlite3. Add file extension as previously asked. 

In your code: if I will create a file cinema_sqlite3_db - it would be some empty file which is not database.